### PR TITLE
VercelDashboard -> Add support for teams with TeamID

### PR DESCRIPTION
**Pull request by MarcusHSmith**
# Description
- Add optional property for `teamId` https://vercel.com/guides/how-do-i-use-a-vercel-api-access-token
- Works for personal account without teamId or with team account with team ID

Screenshot 2024-03-19 at 18 53 07

# Copilot

copilot:all

### CODE REVIEW

**Comment by MarcusHSmith** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MarcusHSmith** [comment]
recheck

File: apps/verceldashboard/vercel_dashboard.star [hunk 1]
```diff
@@ -165,6 +174,12 @@ def get_schema():
                 desc = "Your Vercel API key generated via the Vercel dashboard",
                 icon = "key",
             ),
+            schema.Text(
+                id = CONFIG_TEAM_ID,
+                name = "TEAM ID",
+                desc = "Your Vercel Team ID - required for team deployments. Leave blank for personal deployments.",
```
Comment (croossin) [inline apps/verceldashboard/vercel_dashboard.star:180]: Not sure if you want to put in README or this description but would be good to inform user to ensure the API key has the correct scope for the team as outlined in the API docs

Comment (MarcusHSmith) [reply apps/verceldashboard/vercel_dashboard.star]: Improved description for users

Added additional information to the README including the Configuration

**Pull request merged**